### DISCUSSION TITLE
Fehlermeldungen bei der Übertragung

### DISCUSSION THREAD

**Discussion by fpd62** [GyroGearl00se/ESPHome-Froeling-Lambdatronic_3200#22] [General]
Hallo,

ich habe alles erfolgreich installieren können, habe auch einen 3. und 4. Heizkreis hinzufügen können.
Leider bekomme ich regelmäßig folgende Fehlermeldungen, siehe unten.
Kommt das vom "billigen" RS232 to 3,3V Converter oder von einer schlechten (WiFi) Verbindung? Oder stimmt meine Konfiguration nicht.

Ich finde das Projekt super, es ist genau das, was ich gesucht habe. Die graphische Einbindung kommt noch ...

Vielen Dank!

INFO Successfully connected to froeling @ 172.16.0.75 in 0.037s
INFO Successful handshake with froeling @ 172.16.0.75 in 0.106s
[09:48:53][I][app:102]: ESPHome version 2023.12.9 compiled on Feb 17 2024, 14:20:57
[09:49:17][E][modbus_controller:091]: Modbus error function code: 0x4 exception: 2 
[09:49:17][E][modbus_controller:100]: Modbus error - last command: function code=0x4  register address = 0x7F6B  registers count=1 payload size=0
[09:50:17][E][modbus_controller:091]: Modbus error function code: 0x4 exception: 2 
[09:50:17][E][modbus_controller:100]: Modbus error - last command: function code=0x4  register address = 0x7F6B  registers count=1 payload size=0

**Comment by wavoigt** [comment]
Tritt der Fehler immer bei der gleichen Adresse auf?
Das würde eher nicht auf einen Übertragungsfehler des TTL to 3,3V Converter hinweisen, da wären die Fehler wahrscheinlich zufällig.
Wenn ich in B1200522_ModBus Lambdatronic 3200_50-04_05-19_de.pdf nachschaue, ergibt die Register Adresse 0x7F6B = 32619 - 30001 = 2618 als Adresse.
2618 ist bezeichnet als "Solar - Tagesertrag vor 5 Tagen", das ist jetzt nicht das was du haben willst, nehme ich an.
Hast du vielleicht einen Schreibfehler in deinen yaml Dateien?

**Comment by fpd62** [comment]
Hallo, danke für die schnelle Antwort. Ich war auch gerade bei der Suche nach der Abfrage, konnte aber mit dem Register-Wert nichts anfangen bzw. wusste nicht, das ich die 30001 noch abziehen muss.

Es tritt immer bei der gleichen Abfrage auf, somit scheidet der Wandler (hoffentlich) aus.
Es gibt bei mir zwar die Solarthermie, ich habe habe keinen Wärmemengenzähler aktiv, weder mit einem Durchflusszähler per Impuls oder mit der groben Einstellung Pumpendrehzahl zu Durchfluss. 
Ich vermute mal, der Zugriff wird deshalb dazu geblockt.

Ich nehme diese Abfrage mal raus und schaue, ob es dann ohne Fehlermeldung läuft.

Vielen Dank!

**Comment by fpd62** [comment]
Vielen Dank, das war es. Ich habe die Abfrage nach diesem Wert deaktiviert und schon läuft es stabil und fehlerfrei.
